tagilmo/utils/vereya_wrapper.py

```diff
@@ -263,28 +263,25 @@ def getChat(self, nAgent=0):
         else:
             return None
 
-    def isItemsListAvailable(self, nAgent=0):
-        return not self.observe[nAgent] is None and 'item_list' in self.observe[nAgent]
+    def isParticularObservationAvailable(self, observation_name, nAgent=0):
+        return not self.observe[nAgent] is None and observation_name in self.observe[nAgent]
 
-    def isRecipesListAvailable(self, nAgent=0):
-        return not self.observe[nAgent] is None and 'recipes' in self.observe[nAgent]
-
-    def getItemList(self, nAgent=0):
-        if self.isItemsListAvailable(nAgent):
+    def getParticularObservation(self, observation_name, nAgent=0):
+        if self.isParticularObservationAvailable(observation_name, nAgent):
```

Comment:
this code can be simplified, no need for additional functions:
```suggestion
        obs = self.observe[nAgent]
        if obs is not None:
            return obs.get(observation_name, None)
```

Reply:
@noskill  ok, done.